client/src/ui/BitbakeCommands.ts

```diff
@@ -61,6 +61,14 @@ export function registerBitbakeCommands (context: vscode.ExtensionContext, bitba
       }
     })
   )
+  // Close Toaster on extension shutdown
+  context.subscriptions.push(
+    {
```

Comment:
`push` here accepts an array of disposables. That means one call is enough. But this is not urgent in this MR, I can create another PR for this specific refactor.